app/javascript/components/visualization/controls/CreateAnnotation.jsx

```diff
@@ -352,3 +342,29 @@ function validationMessages(userLabels, annotationName, annotations) {
   }
   return msgs
 }
+
+/** Component for the button for creating an annotiation  */
+function CreateAnnotationButton({ expanded = false, ...props }) {
+  const iconShape = expanded ? faMinus :faPlus
+  const buttonClass = expanded ? 'action-minus' : 'action-plus'
+
+  const toolTipchoice = () => {
+    if (!isUserLoggedIn()) {
+      return 'You must sign in to create custom annotations'
+    } else {
+      if (expanded) {
+        return 'Explore existing annotations'
+      } else {
+        return 'Create custom annotations'
+      }
+    }
+  }
+
+  return <button className = {buttonClass}
+    data-analytics-name = {!isUserLoggedIn() ? 'toggle-create-annotation-signedout' :'toggle-create-annotation'}
```

Comment:
I don't know if it these values were used before, but if not, then using only one seems most conventional.

```suggestion
    data-analytics-name = 'toggle-create-annotation'
```

Regarding convention: a few other elements have different displays when unauthenticated -- e.g. Download button on home page, Download tab in Explore page -- but it seems they all have the same Mixpanel identifier / name whether the user is signed in or not.  Auth state is somewhat of a separate concern, and can be queried via e.g. `authenticated: false` in Mixpanel.

If you can find pre-existing example of this pattern, or you simply think it's worth adopting in certain cases, then the current code works for me!

Reply:
This was the `data-analytics-name` prior to my changes so changing that now would mean needing to go through our existing mixpanel reports to update them and that feels like a separate effort that should be combined with combing through and generally standardizing metric names in the codebase and updating mixpanel reports accordingly. If that seems like a worthwhile effort I can make a ticket for the backlog?